app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -137,4 +138,13 @@ def sso_logging_info
       request_host: request.host
     }
   end
+
+  def can_set_sso_cookie?
+    return Settings.sso.cookie_enabled &&
+            @session_object.present? &&
+            # if the user logged in via SSOe, there is no benefit from
+            # creating a MHV SSO shared cookie
+            @current_user&.issuer !~ /https:\/\/.*eauth.va.gov\/[\S]+/
```

Comment:
We use a slightly different regex in `lib/saml/user.rb#user_attributes_class` so might be good to reconcile those.

It feels a little off to have `issuer` bubble up as an attribute of `User` from an encapsulation perspective, but I think this is largely code that will get refactored/cleaned up as we standardize on SSO, so we can live with it for the time being.

Reply:
I'm open to other ideas if you can think of a better way to send info captured in the saml response to here

Reply:
The only thing that comes to mind is a method like `User.has_va_sso` that can in turn delegate to identity and at least avoids this exact eauth issuer string bubbling up to this class.

Reply:
I don't know if that's meaningfully better 🤔